Formula/openssl@1.1.rb

```diff
@@ -57,11 +56,15 @@ def install
       ENV["PERL"] = Formula["perl"].opt_bin/"perl"
     end
 
-    unless OS.mac?
-      arch_args = %w[linux-x86_64]
-    end
     if OS.mac?
       arch_args = %w[darwin64-x86_64-cc enable-ec_nistp_64_gcc_128]
+    else
+      arch_args = []
+      if Hardware::CPU.intel?
+        arch_args << (Hardware::CPU.is_64_bit? ? "linux-x86_64" : "linux-elf")
+      elsif Hardware::CPU.arm?
+        arch_args << (Hardware::CPU.is_64_bit? ? "linux-aarch64" : "linux-armv4")
```

Comment:
```suggestion
        arch_args << (Hardware::CPU.is_64_bit? ? "linux-aarch64" : "linux-armv6")
```
Is there an `armv6` option? That's the oldest CPU that we support on ARM.

Reply:
Whatever we choose, we should have the same option flag for `openssl`. So if we go for `armv6`, we should update `openssl` in a second PR too later on.

Reply:
I’ll check which arch options are available. You need to query the configure script, it’s not documented anywhere else obvious. I’ll check in a bit and open a PR for `openssl` too if needed.

Reply:
Switched openssl and openssl@1.1 to be `armv6` when 32 bit.